README.md

```diff
@@ -49,6 +49,36 @@ To run YOURLS service with customized config
     $ vim env.yourls
     $ docker compose up [--build] [-d]
 
+## Docker run
```

Comment:
Could you please adjust the section name here and lower it down to `###` but not `##`, and leave a blank line after section title, thanks!

```
### Run with docker compose
...

### Run with docker
...
```